posthog/__init__.py

```diff
@@ -251,6 +251,50 @@ def alias(
     )
 
 
+def capture_exception(
+    exception=None,  # type: Optional[BaseException]
+    distinct_id=None,  # type: Optional[str]
+    properties=None,  # type: Optional[Dict]
+    context=None,  # type: Optional[Dict]
+    timestamp=None,  # type: Optional[datetime.datetime]
+    uuid=None,  # type: Optional[str]
+    groups=None,  # type: Optional[Dict]
```

Comment:
Not sure if `groups` would apply here?

Reply:
why not? if custom exceptions, the caller can / should be able to assign them to groups

Reply:
mm, it's a good point, I just have not seen anything like this, maybe that's the equivalent of what we'd call "tags" elsewhere? would you be able to filter exceptions per groups too?

Reply:
Definitely possible. Since they are eventually stored as events, (and since js automatically adds this info to all events) this will allow for seeing exceptions over time per group (not just persons)

Reply:
awesome